docs/yaml/elementary/str.yml

```diff
@@ -231,6 +231,24 @@ methods:
         Before Meson *0.60.0* this function only accepts a single positional
         argument of the type [[list[str]]].
 
+# str.parse_as_list()
+- name: parse_as_list
+  returns: list[str]
```

Comment:
Comment is redundant.

Needs to have a `since: xxx`

Reply:
Note that all other methods have a similar redundant comment. I can still remove it if you want though.

Reply:
Yeah it seems like those came from the initial checkin, but it's not obvious why they're necessary... I don't strongly care one way or the other I guess.